src/catalog/catalog-flat.md

```diff
@@ -13,71 +14,71 @@ Before you enable a flat catalog for a live store, make sure to test the configu
 
-    * Set **Use Flat Catalog Product** to “Yes.”
+    - Set **Use Flat Catalog Product** to `Yes`.
 
     ![]({{ site.baseurl }}{% link images/images/config-catalog-catalog-storefront-use-flat-catalog.png %}){: .zoom}
-    *Flat Catalog Configuration*
+    _Flat Catalog Configuration_
 
-1. When complete, tap <span class="btn"> Save Config </span>.
+1. When complete, click <span class="btn">Save Config</span>.
 
-1. When prompted to update the cache, click the **Cache Management** link in the system message, and follow the instructions to refresh the cache.
+1. When prompted to update the cache, click the **Cache Management** link in the system message and follow the instructions to refresh the cache.
 
 ## Step 2: Verify the Results
 
 **Method 1: Verify the Results for a Single Product**
 
-   1. On the Admin sidebar, tap **Products**. Then, choose **Categories**.
+   1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
 
-       * Open a product in edit mode.
+   1. Open a product in edit mode.
 
-       * In the **Name** field, add the text “_TEST” to the end of the product name.
+   1. In the **Name** field, add the text `_TEST` to the end of the product name.
 
-   2. Tap <span class="btn"> Save </span>.
+   1. Click <span class="btn"> Save </span>.
 
-   3. On a new browser tab, navigate to the home page of your store. Then, do the following:
+   1. On a new browser tab, navigate to the home page of your store. Then, do the following:
 
-       * Search for the product you edited.
+       - Search for the product you edited.
 
-       * Use the navigation to browse to the product under its assigned category.
+       - Use the navigation to browse to the product under its assigned category.
 
         If necessary, refresh the page to see the results. The change will appear within the minute, or according to your [Cron]({{ site.baseurl }}{% link system/cron.md %}) schedule.
 
         ![]({{ site.baseurl }}{% link images/images/storefront-flat-catalog-enabled.png %}){: .zoom}
-        *Storefront with Flat Catalog*
+        _Storefront with Flat Catalog_
 
 **Method 2: Verify the Results for a Category**
 
-   1. On the Admin sidebar, tap **Products**. Then, choose **Categories**.
+   1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
 
-   2. In the upper-left corner, verify that **Store View** is set to “All Store Views.”
+   1. In the upper-left corner, verify that **Store View** is set to `All Store Views`.
 
-        If prompted, tap **OK** to confirm.
+        If prompted, click **OK** to confirm.
 
-   3. In the category tree, select an existing category. Then, tap <span class="btn"> Add Subcategory </span>, and do the following:
+   1. In the category tree, select an existing category. Then, click <span class="btn">Add Subcategory</span> and do the following:
 
-       * In the **Category Name** field, enter “Test Category.”
+       - In the **Category Name** field, enter `Test Category`.
```

Comment:
These sub-steps aren't numbered or alpha but are in a sequence.

Reply:
The theme doesn't support alpha, like the old UG CSS. Bullets under numbered steps are a workaround for now, but need to consider restructuring many procedures overall.